flask_common/mongo/fields/crypto.py

```diff
@@ -51,3 +52,18 @@ def to_python(self, value):
 
     def to_mongo(self, value):
         return value and self._encrypt(value) or None
+
+
+class EncryptedStringField(EncryptedBinaryField):
+    """
+    Encrypted Unicode string field. Encryption is completely transparent
+    to the caller as the field automatically decrypts when the field
+    is accessed and encrypts when the document is saved. The
+    underlying algorithm currently is AES-256.
+    """
+
+    def to_python(self, value):
+        return value and self._decrypt(value).decode('utf-8') or None
```

Comment:
I know it's been that way here but I prefer an actual ternary `process(value) if value else None` over `value and process(value) or None` as ternary is much more readable and clear in what the intention is.